mkdocs.yml

```diff
@@ -52,6 +52,9 @@ nav:
           - examples/cockroachdb.md
           - examples/nginx.md
           - examples/redis.md
+    - System Requirements:
```

Comment:
Could you remove the entries in the features section, to avoid duplication?

Reply:
Done